README.md

```diff
@@ -32,6 +32,8 @@ your answer to this markdown file.
 Invariant: f(n - 1) = f(n - 2) + f(n - 3)
 
 Reasoning:
-Let f(n) be the nth term of the Fibonnacci sequence.
+At the beginning of each recursive call we can say that for any n > 3, fib(n - 1) can be written as fib(n - 2) + fib(n - 3), as well as fib(0) = 0 and fib(1) = 1.
```

Comment:
What do you mean by "can be written as"? It sounds like you're just giving the definition -- do you want to say something about what the values are as well?